spec/chitter_feed_spec.rb

```diff
@@ -0,0 +1,15 @@
+describe '.content' do
+  it 'returns the chitter feed content' do
+    connection = PG.connect(dbname: 'chitter_test')
+
+    connection.exec("INSERT INTO feed (content) VALUES ('This is message by Luke');")
+    connection.exec("INSERT INTO feed (content) VALUES ('This is another message by Steve');")
+    connection.exec("INSERT INTO feed (content) VALUES ('This is the Chitter Challenge');")
+
```

Comment:
I like that you are adding more than one value, so the tests are accurate.

Reply:
Thank you yeah It good to test multiple things